_docs/geyser/commands.md

```diff
@@ -17,6 +17,7 @@ title: Commands
 | `geyser statistics` | `geyser.command.statistics` | Open the Java statistics menu. |
 | `geyser version` | `geyser.command.version` | Shows the current Geyser version and checks for updates. |
 | `geyser tooltips` | `geyser.command.tooltips`| Toggle showing Advanced Tooltips (F3 + H on Java) |
+| `none` | `geyser.settings.server`| Allow players to use the gameplay menu. (Also requires player to have op.) |
```

Comment:
Not sure about the `none` part.. I'd rather have a separate entry. If possible, I'd all include a screenshot so it's clear what menu is implied here

Reply:
I'd make a new `Other Permissions` section under the table; since the table is for commands imo.
As for image - You can upload an image [here](https://github.com/GeyserMC/GeyserWiki/tree/master/img/wiki), and then link to it using [this](https://github.com/GeyserMC/GeyserWiki/blob/master/_docs/geyser/playit-gg.md?plain=1#L16)

Reply:
I think image is not working, google it but nada.

Reply:
Image shows up fine for me when running a local copy of the wiki; but the page looks kinda... weird with the image and the squished titles. If it's alright with you, i'd like to try and make some small changes that change the look of the page to be more organized

Reply:
Thanks for helping me out!